media/html/visjsDebuggerPanel.html

```diff
@@ -66,6 +66,15 @@
         network = new vis.Network(container, data, message.options);
         vscode = acquireVsCodeApi();
         canvas = document.querySelector('#visjs-network canvas');
+
+        network.on('click', function (properties) {
+          const isCtrlClicked = properties.event.srcEvent.ctrlKey;
+          const ids = properties.nodes;
+          if (isCtrlClicked && ids?.length === 1 && vscode) {
```

Comment:
```suggestion
          if (isCtrlClicked && ids?.length === 1) {
```
`vscode` is always defined at this point (see line 67).